### ISSUE TITLE
Bad default template path

### ISSUE THREAD

**Issue by nickjer** [OSC/ood-myjobs#220]
For a default install of MyJobs I see this when I try to create a new template:

![image](https://user-images.githubusercontent.com/4260509/26839391-e62a25a4-4ab0-11e7-82b6-0d888c922ecf.png)

This is not a valid template directory, and it will copy over the entire MyJobs app (excluding what is in the `.gitignore`) when I create it.

**Comment by nickjer** [comment]
Maybe this should be blank if there are no templates directory?

**Comment by brianmcmichael** [comment]
There's a default template in the app. The problem here was that creating `Template.new ""` sets a `Pathname.new ""` object. When we `expand_path` on that object it uses the app root.

```
irb(main):001:0> p = Pathname.new ""
=> #<Pathname:>
irb(main):002:0> p.expand_path
=> #<Pathname:/users/appl/bmcmichael/ondemand/dev/myjobs>
```

Resolved this with #224
